.github/workflows/ci.yml

```diff
@@ -17,7 +18,7 @@ jobs:
       - id: docker-org-and-tag
         uses: usdot-fhwa-stol/actions/docker-org-and-tag@main
         with:
-          docker_tag_suffix: humble
```

Comment:
We need all the original changes for all the files in this PR. Basically we don't need this PR.
Was something not working?